### PR TITLE
Absorbs Dependency into the core toolset

### PR DESCRIPTION
**Pull request by robmen**
Partially fixes wixtoolset/issues#5945

### CODE REVIEW

File: src/ca/dependencyca.vcxproj [hunk 1]
```diff
@@ -39,30 +39,43 @@
...
+
   <ItemGroup>
-    <None Include="packages.config" />
     <None Include="wixdepca.def" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="WixToolset.Dutil">
+      <Version>4.0.62</Version>
+    </PackageReference>
```
Comment (rseanhall) [inline src/ca/dependencyca.vcxproj:68]: I guess my script needs to handle a new format...

Comment (robmen) [reply src/ca/dependencyca.vcxproj:68]: Heh, yeah, I suppose. I discovered `PackageReference` worked in .vcxproj and it greatly cleans up the project file. Fewer places to update in the extension projects too.

Comment (rseanhall) [reply src/ca/dependencyca.vcxproj:68]: If it's standard, then did you try specifying `Version` as an attribute instead of a child element?

Comment (robmen) [reply src/ca/dependencyca.vcxproj:68]: I thought I did and it didn't work... but it DOES! Pushing that change shortly.

**Comment by rseanhall** [comment]
If you add `<IncludeSymbols>true</IncludeSymbols>` as a property in the extension's .csproj, it should create the .symbols.nupkg as well. Otherwise, the .pdbs are in the main package and that makes it 4MB instead several hundred KB.

**Comment by robmen** [comment]
> If you add <IncludeSymbols>true</IncludeSymbols> as a property in the extension's .csproj, it should create the .symbols.nupkg as well.

I tried that but it seemed to just create a duplicate .symbols.nupkg, both 4MB, identical stuff. Maybe I have something misconfigured.

**Comment by rseanhall** [comment]
That's unfortunate. It magically worked with a handcrafted .nuspec.

**Comment by robmen** [comment]
Frustrating, I can get a .snupkg to have only the .pdbs but they are still present in the .nupkg. There's got to be some trick I'm missing.

**Comment by rseanhall** [comment]
I started trying to figure it out, and I'm pretty sure I already tried this way back when I enabled it for the Bal extension. The problem is that when packing through the SDK-style project, the msbuild task that implements the packing is receiving all of the files that were specified manually and always including those. The file extension filters are only used for files that are automatically included. There's no way to specify something should be only in one of the packages, other than those file extension filters. (https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets)

When packing through an explicitly authored .nuspec, the file extension filters are used on all files equally. I just gave up and manually authored the .nuspec, since we don't want most of the automatic stuff like dependencies and default folders anyway. Plus it has to be manually authored for our packages build from native projects like dutil.

**Comment by robmen** [comment]
Massively frustrating but I too gave up and just made it easier to build .nupkg/.snupkg by pushing the necessary targets into infrastructure. Oh well.

**Pull request merged**